docs/source/gbd2019_models/causes/measles/index.rst

```diff
@@ -0,0 +1,398 @@
+.. _2019_cause_measles:
+
+=======
+Measles
+=======
+
+Disease Description
+-------------------
+
+Measles_ is a highly contagious, serious disease caused by the measles virus
+(*Measles morbillivirus*). Symptoms usually develop 10-12 days after exposure to
+the virus, and last 7-10 days. Symptoms include fever, cough, runny nose,
+conjunctivitis, characteristic white spots inside the cheek (called Koplik's
+spots), and a red, flat, blotchy skin rash that develops on average 14 days
+after exposure to the virus (range, 7-21 days) and lasts 5-6 days. Recovery from
+measles confers lifelong immunity. [WHO]_, [CDC]_, [Wikipedia]_,
+[GBD-2017-YLD-Capstone-Appendix-1]_
```

Comment:
Looks like these are going to give us "duplicate citation" warnings and probably end up linking to the citations on a different page (like the 2017 Measles page). Should we rename the citations to be specific to this page?  Also the 2017 appendix should be updated to the 2019 appendix.  This could all be done in a separate PR, but just thought I'd point it out.

Reply:
Thanks for pointing this out. I don't think that the duplicate citations bring you to another page, but I will update to the 2019 appendix in the other open PR I have for measles that adds detail on the GBD modeling strategy.